docs/reference/esql/esql-syntax.asciidoc

```diff
@@ -36,6 +36,65 @@ source-command | processing-command1 | processing-command2
 ----
 ====
 
+[discrete]
+[[esql-identifiers]]
+==== Identifiers
+
+Unless containing special characters, the identifiers can be used as-are and
+don't require quoting. If they do, however, they must be quoted with
+backticks (```). The special characters are those that have already a semantic
+meaning for a given command, like arithmetic operators (`-`, `+` etc.),
+decimals delimiters (`.`), field delimiters (`,`), command separators (`|`),
+subfield referencing (`.`) a.s.o. This is also applicable to identifiers that
```

Comment:
`a.s.o.`? Maybe provide all of them?

Reply:
I've reworked this paragraph to simplify it.